src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -208,6 +210,24 @@ public boolean equals(Object o)
     User getUserById(String id);
 
     /**
+     * Gets all {@link Guild}s that contain all given users as their members.
+     *
+     * @param users The users which all the returned {@link Guild}s must contain.
+     *
+     * @return Unmodifiable list of all {@link Guild} instances which have all {@link User}s in them.
```

Comment:
Missing `@throws` with full qualified name and newline description
Include the plural `s` in the alias name for the entity that you are linking `Users`